### PR TITLE
Resolves Issue #32

### PR DESCRIPTION
**Pull request by kf6nux**
See issue #32 for details.

### CODE REVIEW

File: request.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func (c *Client) RawRequest(verb, p string, ro *RequestOptions) (*http.Request,
 
 	// Append the path to the URL.
 	u := *c.url
-	u.Path = path.Join(c.url.Path, p)
+	u.Path = strings.TrimRight(c.url.Path, "/") + "/" + strings.TrimLeft(p, "/")
```
Comment (sethvargo) [inline request.go]: As far as I can tell, this preserves the current behavior. If I read your issue correctly, you don't want to alter `p` at all, so I think this needs to become:

```golang
u.Path = strings.TrimRight(c.url.Path, "/") + "/" + p
```

**Comment by sethvargo** [comment]
It would also be great to get a test for this change, to prevent a regression in the case of a fix for #14 in the future.

File: request.go (hunk 1 above)
Comment (kf6nux) [inline request.go]: This preserves the current behavior _only at the join location_.  `p` remains unchanged.

**Comment by kf6nux** [comment]
Tests have been added and the existing Purge test has been fixed.

**Pull request merged**